test/gtest/ucp/test_ucp_wfe.cc

```diff
@@ -0,0 +1,102 @@
+/**
+* Copyright (C) Mellanox Technologies Ltd. 2001-2014.  ALL RIGHTS RESERVED.
```

Comment:
2020

Reply:
We only bump the date if the file was update by the vendor in this particular year.

Reply:
> We only bump the date if the file was update by the vendor in this particular year.

@shamisp thanks! I see